### PR TITLE
Labels

### PR DESCRIPTION
**Pull request by oualib**
Adding labels for each query.

TODO:
 - add label for CREATE and COMMIT queries.
 - add the identifier at the beginning of each method
 - update the contribute file

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     packages=setuptools.find_packages(),
     python_requires=">=3.6",
     install_requires=[
-        "matplotlib>=2.0",
+        "matplotlib<=3.5.3",
```
Comment (afard) [inline setup.py:36]: I think you do this to make the tests pass on Python 8. IMO, it is fine as temporary solution, but we should improve our tests sooner or later to make them resistant to the internal changes of matplotlib.

File: verticapy/tests/vDataFrame/test_vDF_utilities.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 from math import ceil, floor
 
 # Other Modules
-import pandas, geopandas
+# import pandas, geopandas
```
Comment (afard) [inline verticapy/tests/vDataFrame/test_vDF_utilities.py:22]: I wonder what is the reason of relocating the imports?

**Comment by afard** [comment]
I'm curious what is your goal of labeling all the queries?

**Comment by afard** [comment]
There is a library issue on Python 3.6 now. Can you please take a look?

File: setup.py (hunk 1 above)
Comment (oualib) [reply setup.py:36]: I agree

File: verticapy/tests/vDataFrame/test_vDF_utilities.py (hunk 2 above)
Comment (oualib) [reply verticapy/tests/vDataFrame/test_vDF_utilities.py]: It was because of the Python 3.6 bug

**Comment by oualib** [comment]
@afard we are labelling the queries to be able to run statistics on the query_profile table. It will help us to understand the usage of each function.

**Review by afard — APPROVED** [review]

**Pull request merged**